taskcluster/scripts/build/linux.sh

```diff
@@ -43,6 +43,10 @@ while [[ $# -gt 0 ]]; do
   esac
 done
 
+## For debug purposes, dump our environment
+echo "Dumping environment..."
+env
```

Comment:
I think this can be removed right? 
Btw, a tasks "full task definition" always contains the env as well :) 
https://firefox-ci-tc.services.mozilla.com/tasks/EeYFAoBySuiqeW0jiwELdQ/definition

Reply:
Oops! Thanks for catching this.